oms/src/main/java/com/gfg/oms/consignment/request/Item.java

```diff
@@ -0,0 +1,33 @@
+package com.gfg.oms.consignment.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Item {
+    @JsonProperty("simple_sku")
+    private String simpleSku;
```

Comment:
Would be better to put NotNull checks for fields which is required.

Reply:
Sure, added NotNull's everywhere where I think it should be (please check)